From: Bhavna Sarathy <bnagendr@redhat.com>
Date: Thu, 5 Aug 2010 18:04:46 -0400
Subject: [edac] amd64_edac: minor formatting fix
Message-id: <20100805181028.6566.58962.sendpatchset@localhost.localdomain>
Patchwork-id: 27424
O-Subject: [RHEL5.6 PATCH 10/10] amd64_edac - Minor formatting fix
Bugzilla: 568576
RH-Acked-by: Mauro Carvalho Chehab <mchehab@redhat.com>

>From 7106b1576a3290bf83c9449f25d77abcf975ba26 Mon Sep 17 00:00:00 2001
From: Borislav Petkov <borislav.petkov@amd.com>
Date: Wed, 4 Aug 2010 02:54:45 +0200
Subject: [PATCH] amd64_edac: Minor formatting fix

EDAC MC3: CE page 0xc32281, offset 0x8a0, grain 0, syndrome 0x1, row 2, channel 1, label "": amd64_edac
EDAC MC3: CE - no information available: amd64_edacError Overflow

Add the missing space before "Error Overflow" on the second line.

Signed-off-by: Borislav Petkov <borislav.petkov@amd.com>

diff --git a/drivers/edac/amd64_edac.c b/drivers/edac/amd64_edac.c
index 6edbe02..c9ea719 100644
--- a/drivers/edac/amd64_edac.c
+++ b/drivers/edac/amd64_edac.c
@@ -1947,7 +1947,7 @@ static inline void __amd64_decode_bus_error(struct mem_ctl_info *mci,
         * catastrophic.
         */
        if (info->nbsh & K8_NBSH_OVERFLOW)
-               edac_mc_handle_ce_no_info(mci, EDAC_MOD_STR "Error Overflow");
+		edac_mc_handle_ce_no_info(mci, EDAC_MOD_STR " Error Overflow");
 }
 
 void amd64_decode_bus_error(int node_id, struct err_regs *regs)
